.github/workflows/icon-generation.yaml

```diff
@@ -0,0 +1,67 @@
+name: Update Icons
+
+on:
+  workflow_dispatch:
```

Comment:
let’s add a cron job every week just in case there are changes without us being aware?

Reply:
sounds good, will add it